### ISSUE TITLE
Error page: missing level 1 heading and decorative image

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1972]
## Bug Report

**Problematic behavior**
The 503 error page does not have a level 1 heading and the image is decorative.

**Expected behavior/code**
Replace `h3` with `h1` and add the `aria-hidden="true"` attribute on the `svg` element

**Additional context/Screenshots**
<img width="1114" height="726" alt="Image" src="https://github.com/user-attachments/assets/2a9c803d-7ec3-488c-90e1-154194ee0908" />

**Critères RGAA**

[1.2 Chaque image de décoration est-elle correctement ignorée par les technologies d’assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#1.2)

[9.1 Dans chaque page web, l’information est-elle structurée par l’utilisation appropriée de titres ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#9.1)

**Impact**
Decorative images should not be rendered to screen readers. This generates "noise" and can also slow down navigation by rendering unnecessary information.
This can also add cognitive overload for people who use speech synthesis.

Screen reader users can navigate from heading to heading, which allows them to understand the structure of the page but also to orient themselves on the page even if it is particularly empty, giving them the most important information right away.

**Comment by AntoLC** [comment]
This page is not managed in Docs, it appears in rare case when the host has a problem I think.
Developers cannot update this page.

**Comment by virgile-dev** [comment]
Same reply as here https://github.com/suitenumerique/docs/issues/1975#issuecomment-4048440279

**Comment by robin-lecomte** [comment]
<img width="3594" height="1364" alt="Image" src="https://github.com/user-attachments/assets/d860b505-9540-45be-b942-db5758b0ab9b" />

I’ve submitted a proposal for the 503 page.  

I also took the opportunity to suggest an update for all similar pages (500, 404, access denied, email confirmation), ensuring consistent design across LaSuite and introducing new shared visuals.  

[FIGMA](https://www.figma.com/design/qdCWR4tTUr7vQSecEjCyqO/Docs?node-id=13684-20946&t=7pZ9YI0OP2MoRMlP-1)

**Comment by AntoLC** [comment]
I will close it, as we cannot do anything about it from this repository.
